proposals/all_proposals.md

```diff
@@ -0,0 +1,658 @@
+See also https://mobile.twitter.com/henryfarrell/status/1011315278741954560
+
+- platform companies like Facebook or Google need to be treated along the rules of  surveillance capitalism
+
+They provide services to individuals to serve those individuals up to advertisers. But if 
+they cannot demand personal data from their users as a condition, then they don't have anything valuable to provide to their real customers any more. See Max Schrems cases against Facebook.
+
+- platform companies like AirBnb, Uber, Booking etc need different treatment. They  are  collecting data as well but have in many cases severe implications for local economies.  Like AirBnb reduces available living or housing space thus raising the costs of living considerably.
+
+Additional regulation will be required to curb negative effects of this sort of platforms for local economies and the living costs of local people. Taxation of this sort of enterprises must be raised
+to levels of small local firms.
+
+<lynX> Yes, but how?
+
+Technology and labor
+
+Cross references to the respective contents of the European New Deal: SHARING THE RETURNS TO CAPITAL: In the increasingly digital economy, capital goods are increasingly produced collectively but their returns continue to be privatised. As Europe becomes more technologically advanced, to avoid stagnation and discontent it must implement policies for sharing amongst all its citizens the dividends from digitization and automation. Other important issues are:
+
+- weekly  and daily work time reduction 
+- current EU rules require maximum of 48 hours per week averaged over a period of 17  weeks suggestions: max 10 hours per day; max 40 hours per week averaged over 10  weeks; regular work time reduced to 7 hours per day and 35 hours per week (???) all with  full wage compensation
+- reduce online availability esp. of white collar workers
+- support creation of unions or other representation of Single Person Businesses; 
+develop  social networks like unemployment insurance, pregnancy leave, health  insurance, pension insurance
+
+Female Tech, Woman in Technology, Diversity
+
+Women are grossly underrepresented in tech jobs. At least in western countries. However in countries of the former socialist block there are many more women graduating from technical studies as well as working in tech jobs. This is reporting amongst others by dy the study dean 
+of the faculty of informatics at the Technical University of Vienna Hilda Tellioglu and Maria Zesch former CEO of T-Mobile Croatia. Part of the reason might be inadequate methods of teaching especially in mathematics. I know of Bostonian families who send their kids to schools teaching mathematics with methodology and curricula imported from Russia.
+
+
+
+I am not aware of studies about the reasons of the gender gap in tech. If there are none 
+
+- studies need to be carried out to find out the reasons in the teaching methods for the  gender gap in tech
```

Comment:
there are a number of studies, and teaching methods don't appear to be the main issue - this proposal requires some follow-up - kate